src/pages/Components/Notification/previews/notification-states.preview.scss

```diff
@@ -0,0 +1,18 @@
+@import 'settings-tools/_all-settings';
+@import 'components/_c.notification';
+
+body {
+  padding-bottom: 100px;
+}
+
+.example {
+  margin: $mu150;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+}
+
+.example > .mc-notification {
+  margin-top: 25px;
+}
```

Comment:
we don't use pixel but magicUnits

Reply:
I thought it was necessary since this code refers to the style guide only, and the Mozaic itself.